package.json

```diff
@@ -128,7 +129,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-push": "yarn lint --no-fix"
+      "pre-push": "lint-staged"
```

Comment:
Shouldn't `lint-staged` be a script?

Reply:
Oh, so by doing `npx mrm lint-staged` then it should be fine right?

But do we all need to do this in our machines in order for it to work?

Reply:
the package is already installed: https://github.com/luniehq/lunie/pull/3841/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R114

Reply:
Did you try this? Making a push with this?

Reply:
yep, works. to test just fork, change a file (disabling vscode auto format before) and push

Reply:
It's fine, I believe you 😉